src/chain/tezos/TezosNodeWriter.ts

```diff
@@ -226,8 +231,8 @@ export namespace TezosNodeWriter {
         const blockHead = await TezosNodeReader.getBlockHead(server);
         const forgedOperationGroup = forgeOperations(blockHead.hash, operations);
         const signedOpGroup = await signOperationGroup(forgedOperationGroup, keyStore, derivationPath);
-        const appliedOp = await applyOperation(server, blockHead.hash, blockHead.protocol, operations, signedOpGroup);
-        checkAppliedOperationResults(appliedOp);
+        const appliedOp = await preapplyOperation(server, blockHead.hash, blockHead.protocol, operations, signedOpGroup);
+
```

Comment:
Why are we removing this check?

Reply:
That's not a great error validation function, we should make a new one, that's already in progress